pkgs/applications/science/electronics/kicad/packages/kikit.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, bc
+, zip
+, python3
+, package
+, thirdPartyPackagePath
+}:
+let
+  python = python3.withPackages (ps: with ps; [ click ]);
+  kikit = python3.pkgs.kikit;
+  targetSpecs = {
+    "kikit" = {
+      makeTarget = "pcm-kikit";
+      resultZip = "pcm-kikit.zip";
+      description = "KiCAD plugin and a CLI tool to automate several tasks in a standard KiCAD workflow";
+    };
+    "kikit-library" = {
+      makeTarget = "pcm-lib";
+      resultZip = "pcm-kikit-lib.zip";
+      description = "KiKit uses these symbols and footprints to annotate your boards (e.g., to place a tab in a panel).";
+    };
+  };
+  targetSpec = targetSpecs.${package};
+in
+stdenv.mkDerivation {
+  name = "kicadpackage-${package}";
+  inherit (kikit) src version;
+
+  nativeBuildInputs = [ python bc zip ];
+  propagatedBuildInputs = [ kikit ];
```

Comment:
then we can avoid this which I think only works by coincidence

Reply:
We are pretty sure this is necessary for the KiCad python interpreter to be able to import KiKit at runtime. See our comment above for more information.

Reply:
Marking resolved. Please reopen if you have any questions.